crates/re_space_view_spatial/src/lib.rs

```diff
@@ -75,3 +77,57 @@ fn query_pinhole(
                 .map(|c| c.value),
         })
 }
+
+pub(crate) fn background(
+    ctx: &re_viewer_context::ViewerContext<'_>,
+    query: &re_viewer_context::ViewQuery<'_>,
+) -> Option<Background> {
+    let blueprint_db = ctx.store_context.blueprint;
+    let blueprint_query = ctx.blueprint_query;
+    let background = re_space_view::query_space_view_sub_archetype::<Background>(
+        query.space_view_id,
+        blueprint_db,
+        blueprint_query,
+    )
+    .0;
+    background.ok().flatten()
+}
+
+pub(crate) fn configure_background(
+    ctx: &re_viewer_context::ViewerContext<'_>,
+    background: re_types::blueprint::archetypes::Background,
+) -> (Option<re_renderer::QueueableDrawData>, re_renderer::Rgba) {
+    use re_renderer::renderer;
+    use re_types::blueprint::components::BackgroundKind;
+
+    let Background { kind, color } = background;
+
+    match kind {
+        BackgroundKind::GradientDark => (
+            Some(
+                renderer::GenericSkyboxDrawData::new(
+                    ctx.render_ctx,
+                    renderer::GenericSkyboxType::GradientDark,
+                )
+                .into(),
+            ),
+            re_renderer::Rgba::TRANSPARENT, // All zero is slightly faster to clear usually.
+        ),
+
+        BackgroundKind::GradientBright => (
+            Some(
+                renderer::GenericSkyboxDrawData::new(
+                    ctx.render_ctx,
+                    renderer::GenericSkyboxType::GradientBright,
+                )
+                .into(),
+            ),
+            re_renderer::Rgba::TRANSPARENT, // All zero is slightly faster to clear usually.
+        ),
+
+        BackgroundKind::SolidColor => (
+            None,
+            color.unwrap_or(re_types::components::Color::BLACK).into(),
```

Comment:
threw me off that this wouldn't pick a default color from `re_types::blueprint::archetypes::Background`. It's fine but a brief comment would be nice